dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/WorkerGroupServiceImpl.java

```diff
@@ -304,6 +310,8 @@ private List<WorkerGroup> getWorkerGroups(boolean isPaging, List<Integer> ids) {
             if (childrenNodes == null || childrenNodes.isEmpty()) {
                 continue;
             }
+            WorkerGroup aDefault = workerGroups.stream().filter(a -> a.getAddrList().equals("default")).collect(Collectors.toList()).get(0);
```

Comment:
```suggestion
            WorkerGroup aDefault = workerGroups.stream().filter(a -> a.getAddrList().equals("default")).collect(Collectors.toList()).get(0);
```
Please put this out of the for loop